.github/workflows/dev-cluster-check.yml

```diff
@@ -63,6 +64,19 @@ jobs:
           to: openbridge-dev@redhat.com
           from: Dev cluster CI
           body: Dev cluster CI check failed. Please analyze the issue on https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/actions/workflows/dev-cluster-check.yml and implement a proper fix.
+      - name: Get Sheriff Name
```

Comment:
Would it be possible to add the sheriff name also in the body of the email?

Reply:
@r00ta ,
The outcome is not email its a google chat notification.And sheriff github id is displaying.For example,
![image](https://user-images.githubusercontent.com/56147477/170191460-2ffde832-6d0e-4ace-a3fe-d5b64a0aec5d.png).Hope this is fine

Reply:
That's fine. I'm just saying that as we **also** send an email, we could include the sheriff name also there